journal/202104.md

```diff
@@ -0,0 +1,351 @@
+- added a migration to initialize [account](https://github.com/decred/decrediton/pull/2664) passphrases with the wallet passphrase. Also, some operations (send, purchase ticket, mixing, revoke) switched to only unlock a single needed account instead of unlocking an entire wallet.
+- lock [new accounts](https://github.com/decred/decrediton/pull/3424) after creation
+- allow restoring hex seeds up to 128 characters (including BIP0039 seeds)
+- updated design of [Transactions](https://github.com/decred/decrediton/pull/3326), [Proposals](https://github.com/decred/decrediton/pull/3345) and [Purchase tickets](https://github.com/decred/decrediton/pull/3349) views
+- many smaller design tweaks
+- switched to a strict Protobuf generation without `eval()` and restricted to only access exteral servers via `https` in production builds using the [CSP header](https://github.com/decred/decrediton/pull/3366)
+- refactoring of state management, separation of concerns, dependency upgrades
+- expanded test coverage
+- ~24 bug fixes
+
+Release [v1.6.3](https://github.com/decred/decrediton/tree/release-v1.6%2B) is being prepared.
+
+<a id="politeia" />
+
+**[Politeia](https://github.com/decred/politeia)**
+
+- list [proposals](https://github.com/decred/politeiagui/pull/2340) from [proposals-archive.decred.org](https://proposals-archive.decred.org/) on the new version
+- [verify](https://github.com/decred/politeia/pull/1377) comment and vote bundles in the `politeiaverify` tool
+- add Unix [timestamp](https://github.com/decred/politeia/pull/1395) to cast votes { is there no temporal batching due to tstore now? does it lower privacy? }
+- list registration fee in the [history](https://github.com/decred/politeiagui/pull/2322) of purchases
+- ~24 bugs discovered in the new version have been fixed
+
+Politeia v1.0.0 has been released including the new tstore backend, v2 API, new plugin architecture and 5 plugins. Read the [release notes](https://github.com/decred/politeia/releases/tag/v1.0.0) for full details on each feature. The new version has been deployed at [proposals.decred.org](https://proposals.decred.org/).
+
+Check the new [proposal](https://proposals.decred.org/record/91cfcc8) to learn about the development roadmap and budget.
+
+<a id="vspd" />
+
+**[vspd](https://github.com/decred/vspd)**
+
+- added [parser](https://github.com/decred/vspd/pull/238) for `blockconnected` notification to remove a dependency
+- upgraded [dependencies](https://github.com/decred/vspd/pull/239)
+- removed an unnecessary [check](https://github.com/decred/vspd/pull/241) as a step to improve [database](https://github.com/decred/vspd/issues/223) performance
+
+<a id="dcrpool" />
+
+**[dcrpool](https://github.com/decred/dcrpool)**
+
+- [improved](https://github.com/decred/dcrpool/pulls?q=is%3Apr+merged%3A2021-04-01..2021-04-30+sort%3Aupdated-asc) connectivity and [miner](https://github.com/decred/dcrpool/pull/319) handling
```

Comment:
I have a feeling the `improved` link was meant to be to a different URL.

Reply:
I used the "all April's PRs" link originally because multiple PRs improved the same aspect. But now that you've mentioned this I've revised it in a way that links to exact PRs.